dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -303,6 +306,7 @@ private static Container getContainerFields(Map<String, String> map) throws Pars
         Container container;
         container = new Container();
         container.setInode(map.get(INODE));
+        container.setOwner(map.get(OWNER));
         container.setCode(map.get("code"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "setCode"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)